languages/ruby/exercises/concept/numbers/.meta/design.md

```diff
@@ -27,6 +27,7 @@ The Concepts this exercise unlocks are:
 This exercise's prerequisite Concepts are:
 
 - `basics`: know how to define methods.
+- `booleans`: know about `true`/`false` and truthy/falsy values.
```

Comment:
```suggestion
- `booleans`: know about `true`/`false` and truthy/falsey values.
```

Reply:
I searched the V3 repo and both `falsy` and `falsey` seem to appear almost equally. We should probably agree on on in the repo